src/components/Callback/Callback.tsx

```diff
@@ -0,0 +1,148 @@
+import { useCallback, useState, useEffect } from 'react';
+import { LegacyTokens, requestLegacyToken, requestOidcToken, OIDCError, OIDCErrorType } from '../../oidc';
+import ErrorIcon from '../../assets/404.svg?react';
+import DerivLogoIcon from '../../assets/deriv_logo.svg?react';
+
+import './Callback.scss';
```

Comment:
let's submit a request in #need_help_design_team for this

Reply:
Yes, we need design requirements for this, for now this is just a placeholder